.travis.yml

```diff
@@ -2,8 +2,7 @@ language: ruby
 before_install:
   - gem install bundler
 rvm:
-  - 2.1.0
-  - 2.0.0
-  - 1.9.3
+  - 2.3.0
+  - 2.4.0
```

Comment:
Rails 5 only supports Ruby 2.2 or higher, so I'm changing the rvm to use 2.2-2.4 versions.